adobe/spectrum-web-components
# fix(progress-bar): removed duplicate label

## Description

Removes duplicate label from progress bar.

## Related issue(s)

- fixes #4225

## How has this been tested?




    1. Go [here](https://nikkimk-4225-bug-progress-label--spectrum-web-components.netlify.app/components/progress-bar/#sizes)
    2. Right click the first progress bar example and choose **Inspect Element**
    3. From the **Element Inspector** click on the progress bar HTML and select **Edit HTML**
    4. Replace the existing progress bar HTML the following HTML: `<sp-progress-bar label="" size="m" indeterminate ></sp-progress-bar>`.
    5. Click **Enter**.
    6. Verify that the new progress bar displayed has no label.
    7. Right click the new progress bar's HTML and choose **Add Attribute**.
    8. Type the attribute `label="label1"`.
    9. Click **Enter**.
    10. Verify that the progress bar label displays as `label1` not `label1label1`.


## Types of changes


## Checklist

-   [x] I have added tests to cover my changes. _(Note: Testing won't allow an empty an empty label, so I can't test for this.)_
